src/networktables/NetworkTable.cpp

```diff
@@ -44,23 +41,41 @@ void NetworkTable::SetClientMode() { s_client = true; }
 void NetworkTable::SetServerMode() { s_client = false; }
 
 void NetworkTable::SetTeam(int team) {
-  char tmp[30];
-#ifdef _MSC_VER
-  sprintf_s(tmp, "roboRIO-%d-FRC.local", team);
-#else
-  using namespace std;
-  snprintf(tmp, 30, "roboRIO-%d-FRC.local", team);
-#endif
-  SetIPAddress(tmp);
+  std::pair<StringRef, unsigned int> servers[3];
+
+  // roboRIO-<team>-FRC.local
```

Comment:
Is there a preferred ordering or are they done in parallel? If ordered, I'd prefer the IP address to be first.